### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by beetai**
and a few more things

### CODE REVIEW

**Review by DSep — APPROVED** [review]
Bless you.

**Review by yqj2k — APPROVED** [review]

**Review by baconandchips — APPROVED** [review]
LGTM!

File: server/routes/auth.js [hunk 1]
```diff
@@ -41,8 +41,9 @@ router.post('/', async (req, res, next) => {
 // get current user data using token: private
 router.get('/user', auth, (req, res, next) => {
   User.findById(req.user.id)
-    .select('-password')
-    .then((user) => res.json(user));
+  .select('-password')
+  .then((user) => res.json(user))
+  .catch((err) => res.status(400).json(err.message));
```
Comment (beetai) [inline server/routes/auth.js:46]: indentation

File: frontend/src/components/Navbar.tsx [hunk 2]
```diff
@@ -112,30 +127,34 @@ function Navbar(props: any) {
...
+                  root: classes.inputRoot,
+                  input: classes.inputInput,
+                  focused: classes.inputFocused,
+                }}
+                inputProps={{ 'aria-label': 'search' }}
+                onChange={handleQueryChange}
+                onKeyDown={handleKeyDown}
+              />
+            </div>
+          )}
```
Comment (beetai) [inline frontend/src/components/Navbar.tsx:157]: wow okay cool

File: frontend/src/components/AccountForm.tsx [hunk 3]
```diff
@@ -156,7 +156,7 @@ function AccountForm(props: any) {
               </Tooltip>
             </ListItem>
           )}
-        <ListItem>
+        {/* <ListItem>
```
Comment (beetai) [inline frontend/src/components/AccountForm.tsx]: remove commented lines?

File: frontend/src/components/LoginForm.tsx [hunk 4]
```diff
@@ -10,11 +10,12 @@ import { login } from '../redux/actions/auth';
 import { LOGIN_FAIL } from '../redux/constants';
 import { clearErrors } from '../redux/actions/error';
 import { clearModals } from '../redux/actions/modal';
+import {FormControl, InputLabel} from "@material-ui/core";
```
Comment (beetai) [inline frontend/src/components/LoginForm.tsx]: lint change

File: frontend/src/components/LoginModal.tsx [hunk 5]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
-import { Fade, Link, Modal, Typography } from '@material-ui/core';
+import {Fade, Modal, Typography, IconButton, Button} from '@material-ui/core';
```
Comment (beetai) [inline frontend/src/components/LoginModal.tsx]: lint change

File: frontend/src/components/Navbar.tsx [hunk 6]
```diff
@@ -112,38 +127,34 @@ function Navbar(props: any) {
       <Toolbar>
         <Typography className={classes.title} variant="h5">
           <Button onClick={() => history.push('/')}>
-            CourseFlo
+            {matchesXs ? 'Flo' : 'CourseFlo'}
```
Comment (beetai) [inline frontend/src/components/Navbar.tsx:130]: cool lol

File: frontend/src/components/Results.tsx [hunk 7]
```diff
@@ -16,7 +16,11 @@ interface Props {
 const Results = (props: Props) => {
   const { searchResults }: Props = props;
 
-  searchResults.sort((a, b) => (a.courseDigitCode > b.courseDigitCode ? 1 : -1));
+  searchResults.sort((a, b) => {
+    if (a.courseDigitCode > b.courseDigitCode) return 1;
+    if (a.courseDigitCode === b.courseDigitCode && a.courseLetterCode > b.courseLetterCode) return 1;
```
Comment (beetai) [inline frontend/src/components/Results.tsx:21]: :sob: alright @DSep

File: frontend/src/components/Search.tsx [hunk 8]
```diff
@@ -190,6 +191,11 @@ const Search = (props: any) => {
                 step={SLIDER_STEP_SIZE}
                 valueLabelDisplay="auto"
                 marks={sliderMarks}
+                // getAriaValueText={valuetext}
```
Comment (beetai) [inline frontend/src/components/Search.tsx]: remove comment?

File: frontend/src/components/SignupForm.tsx [hunk 9]
```diff
@@ -8,11 +8,12 @@ import { register } from '../redux/actions/auth';
 import { REGISTER_FAIL } from '../redux/constants';
 import { clearErrors } from '../redux/actions/error';
 import { clearModals } from '../redux/actions/modal';
+import {FormControl, InputLabel} from "@material-ui/core";
```
Comment (beetai) [inline frontend/src/components/SignupForm.tsx]: lint changes

File: frontend/src/components/SignupModal.tsx [hunk 10]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
-import { Fade, Link, Modal, Typography } from '@material-ui/core';
+import {Button, Fade, IconButton, Modal, Typography} from '@material-ui/core';
```
Comment (beetai) [inline frontend/src/components/SignupModal.tsx]: lint changes

File: server/routes/auth.js (hunk 1 above)
Comment (beetai) [inline server/routes/auth.js]: indentation

**Pull request merged**